qemu/tests/cfg/boot_order_check.cfg

```diff
@@ -6,6 +6,9 @@
     kill_vm = yes
     boot_menu = on
     enable_sga = yes
+    Host_RHEL.m9:
```

Comment:
what about defining 
enable_sga = no
machine_type_extra_params = "graphics=off"

then for the previous rhel releases:
HOST_RHEL.m7, m8...
    enable_sga = yes

So when 10/11 comes out, we don't have to add them

Reply:
Thanks for your quickly review. sga not only used for rhel host  but also other linux hosts. It just removed in rhel9, so I just disable it in rhel9. I think we have to add them when 10/11 comes out. Thanks.